source/_integrations/google_sheets.markdown

```diff
@@ -85,3 +85,44 @@ data:
 {% endraw %}
 
 {% enddetails %}
+
+
+### Action `google_sheets.get_sheet`
+
+You can use the `google_sheets.get_sheet` action to retrieve rows of [data] (/docs/scripts/perform-actions#use-templates-to-handle-response-data) from a Sheets document.
+
+{% details "Create event action details" %}
+
+| Data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | --------|
+| `config_entry` | no | Config entry to use. |
+| `worksheet` | yes | Name of the worksheet. Defaults to the first one in the document. | Sheet1 |
+| `rows` | no | Maximum number of the last rows of data to be retrieved from the worksheet.  | 2 |
```

Comment:
Maybe this would work:
"Maximum number of rows from the end of the worksheet to return"